awscli/topics/config-vars.rst

```diff
@@ -476,6 +478,38 @@ To set encoding used for text files different from the locale, you can use
 with default encoding ``CP1252``, setting ``AWS_CLI_FILE_ENCODING=UTF-8`` would
 make CLI ignore locale encoding and open text files using ``UTF-8``.
 
+Pager
+-----
+
+AWS CLI uses pager for output data that don't fit on the screen.
+
+By default if on Linux/MacOS, use ``less`` as the pager on Windows
+by default use ``more``
+
+    You can override the default pager with the following configuration
```

Comment:
I think it may make sense to break some of these paragraphs into their own subsubsection. So for this one, we can label it `Configuring pager` instead of having to rely on block quotes. So:
```
Configuring pager
^^^^^^^^^^^^^^^^^^
You can override the default pager with the following configuration options. The order of precedence is:

* ``AWS_PAGER`` environment variable
* ``cli_pager`` shared config variable
* ``PAGER`` environment variable

```